operator/pkg/helmreconciler/prune.go

```diff
@@ -147,6 +148,13 @@ func (h *HelmReconciler) DeleteObjectsList(objectsList []*unstructured.Unstructu
 					h.opts.Log.LogAndPrintf("object: %s is not being deleted because it no longer exists",
 						obj.Hash())
 				}
+			} else {
+				objGvk := o.GroupVersionKind()
+				metrics.ResourceDeletionTotal.
+					With(metrics.ResourceKindLabel.Value(util.GKString(objGvk.GroupKind()))).
+					Increment()
+				h.addPrunedKind(objGvk.GroupKind())
```

Comment:
why is it needed here?

Reply:
We are deleted an object => we have removed/pruned something. As we are counting number of reconciliations a particular resource kind was pruned, this should be considered. There is a delete on top of it.